lib/smart_proxy_pulp_plugin/disk_usage.rb

```diff
@@ -0,0 +1,37 @@
+module PulpProxy
+  class DiskUsage
+    include ::Proxy::Util
+    SIZE = {:kilobyte => 1_024, :megabyte => 1_048_576, :gigabyte => 1_073_741_824, :terabyte => 1_099_511_627_776}
+
+    attr_reader :path, :stat, :size
+
+    def initialize(opts ={})
+      @path = opts[:path] || PulpProxy::Plugin.settings.pulp_dir
```

Comment:
if we scan for multiple dirs (mongo, pulp and pulp/content) i guess this should not be the default.